layouts/index.html

```diff
@@ -28,19 +22,13 @@
         {{ $longDesc }}
       </p>
 
-      <!-- <br />
-
-      <div class="notification has-background-white is-radiusless">
-        <p class="title is-size-3 is-size-4-mobile has-text-weight-light">
-          TiKV is now a <a href="https://cncf.io" target="_blank">Cloud Native Computing Foundation</a> incubating project! 🎉🍾
+      <div class="notification is-radiusless">
+        <p class="title is-size-3 is-size-4-mobile">
+          TiKV is a <a href="https://cncf.io" target="_blank" class="is-link">Cloud Native Computing Foundation</a> graduated project
```

Comment:
```suggestion
          We are a <a href="https://cncf.io" target="_blank" class="is-link">Cloud Native Computing Foundation</a> graduated project.
```

Reply:
Can we add some cheering emojis? @YiniXu9506

Reply:
Yes. How about this? 🎉

Reply:
Looks good!